internal/configs/config_params.go

```diff
@@ -196,8 +196,8 @@ func NewDefaultGlobalConfigParams() *GlobalConfigParams {
 func NewGlobalConfigParamsWithTLSPassthrough() *GlobalConfigParams {
 	return &GlobalConfigParams{
 		Listeners: map[string]Listener{
-			conf_v1alpha1.TLSPassthroughListenerName: {
-				Protocol: conf_v1alpha1.TLSPassthroughListenerProtocol,
+			conf_v1.TLSPassthroughListenerName: {
```

Comment:
were these lines covered by a unit test earlier?

Reply:
No, these never had unit tests.

Reply:
Should we address the Codecov warnings? Is that viable?

Reply:
@danielnginx a lot of that code can't be coverd by unit tests, or it doesn't make sense to unit test it